cmd/ingest/main.go

```diff
@@ -158,7 +161,14 @@ func Main() error {
 		}
 	}()
 	var g run.Group
-	if err := runGroup(ctx, &g, q, appFlags, c, logger, reg); err != nil {
+	sources, destinations, err := c.ConfigurePlugins(ctx, *appFlags.pluginDirectory)
```

Comment:
doesn't really matter, I just wanted to simplify the PR :)

Reply:
actually we need to exit before that because we would want to exit before we configure the queue. Right now dry run would fail if NATS is not running. I will make the change.

Reply:
Ah nice! That makes a lot of sense!